### ISSUE TITLE
A11Y issue on https://plone.org/services/providers due to title tag

### ISSUE THREAD

**Issue by ichim-david** [plone/plone.org#181]
When I hover over any of the [Plone providers](https://plone.org/services/providers) I get the title "Foundation Sponsor card" due to this title being applied on the parent div as seen in this screenshot.

![plone-foundation-sponsor-card](https://github.com/user-attachments/assets/f0ba50e1-19bf-4c26-9dfa-a4e6285726c7)

This title should  be removed to avoid wrong information given both to users that hover over the logo's and those that use a screen reader to navigate and hear Foundation Sponsor Card when this listing is concerning plone providers

**Comment by ichim-david** [comment]
@stevepiercy how come I have no permission to add any labels to this issue? 
I see no reason for the Plone developers who are part of this team 
https://github.com/orgs/plone/teams/developers
should not be allowed to add labels to issues added on plone.org which is part of the plone organisation.

**Comment by stevepiercy** [comment]
@ichim-david I see the same issue on:

- https://plone.org/foundation/sponsorship/sponsors

I also added the URL to the page which you initially reported.

---

This repo does not have the Developers team assigned to it. It does have [Website-Devs](https://github.com/orgs/plone/teams/website-devs), whose description is "Developers working on our Plone websites like plone.org and ploneconf.org" assigned to it, as well as 4 individuals. I'm a member of the Website-Devs team. I can't remember what I did to earn the ~~burden~~ honor.

I've pinged the team and individuals to explain permission management of these repositories. I assume that making a contribution to the repos has nothing to do with making changes to content on ploneconf.org or plone.org, which requires the usual Plone site authorization. If that's true, then it makes sense to allow the Developer team to have write permission to these repos.

@plone/website-devs @mpeeters @nzambello @pnicolli @rioksane would one of you please enlighten us to permissions management for this repo and the `@plone/website-devs` team? Thank you!

**Comment by ichim-david** [comment]
I understand need for protection for the plone.org website, maybe adding labels means special permissions but if not it would be nice for the plone developers to flag a bug as a bug :)

**Comment by stevepiercy** [comment]
Exactly. I want to make the distinction between contributing _code_ versus _content_ to plone.org and other sites. These require two different authorization schemes and frames of mind.
